openfl/component/aggregator/aggregator.py

```diff
@@ -341,8 +341,10 @@ def get_aggregated_tensor(self, collaborator_name, tensor_name,
         #  The returned tensor will
         # be recompressed anyway.
         if 'compressed' in tags:
+            tags = list(tags)
```

Comment:
It looks like it should be always `list` according to openfl/protocols/federation.proto:41.
If convert is really needed maybe we can do it in the start of the fanction.

Reply:
The cast is necessary (an error is thrown about modifying a tuple otherwise) - but I've moved this out of the if statements